cogstat/test/test_stat.py

```diff
@@ -195,20 +195,20 @@ def test_explore_variable_pairs(self):
             # Spearman's correlation: jamovi 2.2.5.0 0.103
         self.assertTrue('<i>r<sub>s</sub></i> = 0.103' in result[5])
             # Slope CI: jamovi 2.2.5.0 [-0.440, 1.13]
-        self.assertTrue('[-0.440, 1.133]' in result[9])
+        self.assertTrue('[-0.440, 1.133]' in result[11])
```

Comment:
Please, add a case where Henze-Zirkler test is checked.

Reply:
Henze-Zirkler test check added.